cluster-state-service/handler/reconcile/loader/taskloader.go

```diff
@@ -148,6 +153,31 @@ func (loader taskLoader) getTasksFromECS(cluster *string) ([]types.Task, error)
 	return tasks, nil
 }
 
+// getTaskARNsFromECS gets a list of the task ARNs from ECS for both running and stopped tasks.
+// ECS currently stores stopped tasks for about 1 hour.
+func (loader taskLoader) getTaskARNsFromECS(cluster *string) ([]*string, error) {
+	taskARNs := make([]*string, 0)
```

Comment:
The make function requires a length parameter when creating slices. It is optional when creating maps or channels.